djangoproject/settings/docker.py

```diff
@@ -1,4 +1,4 @@
-from .common import *  # noqa
+from .dev import *  # noqa
```

Comment:
```suggestion
from .dev import *  # noqa: F403
```
Since you're editing this line, we can add the exact flake8 rule to exclude.

Reply:
Done.